<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>variable_X</class>
 <widget class="QWidget" name="variable_X">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>379</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>276</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>276</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Введи значение переменной Х</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(234, 246, 249)</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>256</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QPushButton" name="button_back">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: rgb(255, 120, 138);
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(255, 79, 91);
}

QPushButton:pressed {
	background-color: rgb(255, 0, 13);
}</string>
      </property>
      <property name="text">
       <string>⌫</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="button_6">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: rgb(255, 235, 238);
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(176, 254, 255)
}

QPushButton:pressed {
	background-color: rgb(87, 252, 255)
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="button_0">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: rgb(255, 235, 238);
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(176, 254, 255)
}

QPushButton:pressed {
	background-color: rgb(87, 252, 255)
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="button_C">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: rgb(255, 120, 138);
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(255, 79, 91);
}

QPushButton:pressed {
	background-color: rgb(255, 0, 13);
}</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="button_4">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: rgb(255, 235, 238);
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(176, 254, 255)
}

QPushButton:pressed {
	background-color: rgb(87, 252, 255)
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="button_9">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: rgb(255, 235, 238);
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(176, 254, 255)
}

QPushButton:pressed {
	background-color: rgb(87, 252, 255)
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="button_7">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: rgb(255, 235, 238);
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(176, 254, 255)
}

QPushButton:pressed {
	background-color: rgb(87, 252, 255)
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="button_8">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: rgb(255, 235, 238);
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(176, 254, 255)
}

QPushButton:pressed {
	background-color: rgb(87, 252, 255)
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="button_ok">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: rgb(35, 141, 255);
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(75, 75, 255)
}

QPushButton:pressed {
	background-color: rgb(22, 1, 255)
}</string>
      </property>
      <property name="text">
       <string>©</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="button_1">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: rgb(255, 235, 238);
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(176, 254, 255)
}

QPushButton:pressed {
	background-color: rgb(87, 252, 255)
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="button_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: rgb(255, 235, 238);
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(176, 254, 255)
}

QPushButton:pressed {
	background-color: rgb(87, 252, 255)
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="button_dot">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: white;
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(176, 254, 255)
}

QPushButton:pressed {
	background-color: rgb(87, 252, 255)
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="button_3">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: rgb(255, 235, 238);
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(176, 254, 255)
}

QPushButton:pressed {
	background-color: rgb(87, 252, 255)
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="button_5">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: rgb(255, 235, 238);
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(176, 254, 255)
}

QPushButton:pressed {
	background-color: rgb(87, 252, 255)
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="button_abs">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: white;
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(176, 254, 255)
}

QPushButton:pressed {
	background-color: rgb(87, 252, 255)
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="monitor">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>256</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>70</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: white;
border: 2px solid black;
border-top-left-radius: 5px;
border-top-right-radius: 5px;
border-bottom-left-radius: 5px;
border-bottom-right-radius: 5px;
color: #999;
font-size: 21pt;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
